packages/ui/react-ui/README.md

```diff
@@ -14,7 +14,20 @@ First, add the Crossmint Client SDK to your project with the following command:
 yarn add @crossmint/client-sdk-react-ui
 ```
 
-The next steps depend on which chain you're using for your project. For detailed integration instructions, visit our documentation for [Solana](https://docs.crossmint.io/accept-credit-cards/integration-guides/solana-candy-machine/b-i-have-an-existing-candy-machine-website/add-crossmint-for-solana-react.js-next.js), [Polygon](https://docs.crossmint.io/accept-credit-cards/integration-guides/polygon), or [Ethereum](https://docs.crossmint.io/accept-credit-cards/integration-guides/ethereum/add-crossmint-for-ethereum-react.js-next.js).
+### I'm using Payments
```

Comment:
I think these should match to what we have in docs, so here: "Add payment button to your site" as in https://docs.crossmint.com/docs/crossmint-pay-button